subjects/mobile-dev/movie-list/README.md

```diff
@@ -1,30 +1,20 @@
-# Movie List
+## Movie List
 
 ### Introduction
 
-When developing a fully functional app, you will most likely use some external files to display on your app. One of the most popular file types is JSON.  
-Develop an app to work with JSON. You are given a file with information about different movies.
-You must display top rated movies with descending rating on the first page of the app. By tapping on a movie, a new route with more detailed information regarding the movie must be displayed.  
-Searching for a movie via search bar must be included. Searching must be done by movie name, Entered string must be completely within movie name, i.e. if "vatar" is searched, "Avatar" must be included in the response. Basically, search must work like SQL's **_ilike_** comparision.
+When developing a fully functional app, you will most likely use some external files to display on your app. One of the most popular file types is `JSON`.
 
-### Objective
+In this exercise, you will develop a fully functional app that uses `JSON` files to display information about movies. You will be given a [movies](movies.json) file with information about different movies.Your app needs to have the following features:
 
-- Infinite scroll
-- ListView
-- Json Serialization
-- Routes/Push
-- AppBar actions
-- Images
-- Futures, async
-- Provider
+- Display the top rated movies with descending rating on the home page of the app.
+- By tapping on a movie, a new route with more detailed information regarding the movie must be displayed.
+- You need to include a `search bar` to search for movies by name. The search function must return movies whose name includes the entered string. For example, if "vatar" is searched, "Avatar" must be included in the response. The search function must work like **SQL's ilike** comparison.
 
-### Part 1
+### ListView
 
-Create a ListView which will show the image, title and description of movies from JSON file (see **Assets** section);
+Create a `ListView` which will show the `image`, `title`, and `genre` of movies from the given [Movies](movies.json) file. To create the `ListView`, you need to create a `Movie class` with the following properties:
```

Comment:
Little detail - `Movies` is with capital letter here, while before was lowercase (in the middle of a sentece)

Reply:
done, thanks